src/libraries/Microsoft.PowerFx.Connectors/ConnectorConstants.cs

```diff
@@ -10,6 +10,7 @@ public static class Constants
         public const string XMsDynamicProperties = "x-ms-dynamic-properties";
         public const string XMsDynamicSchema = "x-ms-dynamic-schema";
         public const string XMsDynamicValues = "x-ms-dynamic-values";
+        public const string XMsEnum = "x-ms-enum";
```

Comment:
> public const string XMsEnum = "x-ms-enum"; [](http://example.com/codeflow?start=7&length=43)

is this part of ADS or something else?

Reply:
nope
we were missing x-ms-enum management which is unrelated to ADS
it's used in tests in DV PR for naming enumerations in swagger